common.blocks/forum-form/forum-form.bemtree.js

```diff
@@ -0,0 +1,61 @@
+block('forum-form').content()(function() {
+    var i18n = this.require('i18n');
+
+    return [
+        {
+            elem: 'forum-create',
+            content: [
+                {
+                    elem: 'header',
+                    content: {
+                        block: 'radio-group',
+                        mods: { theme: 'islands', size: 'm', type: 'button' },
+                        name: 'radio-button',
+                        val: 1,
+                        options: [
+                            { 
+                                val: 1,
```

Comment:
why such strange values? use `markdown` and `preview`. so it'll be much easier to get the value on `change` event instead of modifier usage.